### PR TITLE
style(ui): align peerId and radicleId styling in the app

### PR DESCRIPTION
**Pull request by juliendonck**
- bumps up fontsize to match designs
- always shows copy icon for discoverability and to avoid jumping ui
- add truncation

Screen Shot 2020-11-20 at 15 33 03
Screen Shot 2020-11-20 at 15 33 12
Screen Shot 2020-11-20 at 15 33 16

### CODE REVIEW

**Review by MeBrei — DISMISSED** [review]
Do we also want the same behavior of the peer id/radicle id in the following places. 
- [x] Following tab
Screenshot 2020-11-20 at 15 45 25

- [x] Manage remotes
Screenshot 2020-11-20 at 15 45 47

**Review by cloudhead — APPROVED** [review]
:+1:

**Comment by cloudhead** [comment]
> * Manage remotes

I think this is fine, but we could consider adding the computer icon :shrug:

**Comment by cloudhead** [comment]
>  Following tab

We have the space here to show the full ID, but yeah idk, I'd be fine either way.

**Comment by juliendonck** [comment]
> Do we also want the same behavior of the peer id/radicle id in the following places.
> 
> * [x]  Following tab

It's fine here. That's using the RadicleId component. 

> 
> * [x]  Manage remotes

Used the `<PeerId />` component here and made it non expandable.

File: ui/DesignSystem/Component/RadicleId.svelte [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  export let style = "";
+  export let urn = "";
+  export let truncate: boolean = false;
+
+  const hash = urn.replace("rad:git:", "");
+</script>
+
+<style>
+  .peer-id {
```
Comment (rudolfs) [inline ui/DesignSystem/Component/RadicleId.svelte:13]:
```suggestion
  .radicle-id {
```

File: ui/DesignSystem/Component/RadicleId.svelte [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    display: flex;
+  }
+</style>
+
+<div class="peer-id" {style} data-cy="urn" title={urn}>
```
Comment (rudolfs) [inline ui/DesignSystem/Component/RadicleId.svelte:18]:
```suggestion
<div class="radicle-id" {style} data-cy="urn" title={urn}>
```

**Review by rudolfs — DISMISSED** [review]

**Review by cloudhead — APPROVED** [review]

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**